build.cake

```diff
@@ -724,23 +737,39 @@ Task("samples")
     CleanDirectories(packagesPath);
 
     // build the samples
-    var settings = new MSBuildSettings()
+    MSBuildSettings settings_msbuild = new MSBuildSettings()
         .SetConfiguration(CONFIGURATION)
         .SetVerbosity(VERBOSITY)
         .SetMaxCpuCount(0)
-        .EnableBinaryLogger($"./output/samples.{CONFIGURATION}.binlog")
+        .EnableBinaryLogger($"./output/samples.{CONFIGURATION}.msbuild.{DateTime.Now.ToString("yyyyMMddHHmmss")}.binlog")
         .WithRestore()
         .WithProperty("RestorePackagesPath", packagesPath)
         .WithProperty("DesignTimeBuild", "false")
         .WithProperty("AndroidSdkBuildToolsVersion", $"{AndroidSdkBuildTools}");
 
     if (!string.IsNullOrEmpty(ANDROID_HOME))
-        settings.WithProperty("AndroidSdkDirectory", $"{ANDROID_HOME}");
+        settings_msbuild.WithProperty("AndroidSdkDirectory", $"{ANDROID_HOME}");
 
     if (!string.IsNullOrEmpty(MSBUILD_PATH))
-        settings.ToolPath = MSBUILD_PATH;
-
-    MSBuild("./samples/BuildAll/BuildAll.sln", settings);
+        settings_msbuild.ToolPath = MSBUILD_PATH;
+
+    Information($"=====================================================================================================");
+    Information("MSBuild    ./samples/BuildAll/BuildAll.sln");    
+    MSBuild("./samples/BuildAll/BuildAll.sln", settings_msbuild);
+    Information($"=====================================================================================================");
+    Information("MSBuild    ./samples/BuildXamarinFormsApp/BuildXamarinFormsApp.sln");
+    // MSBuild("./samples/BuildXamarinFormsApp/BuildXamarinFormsApp.sln", settings_msbuild);
```

Comment:
> Are all of these `MSBuild`/`DotNetBuild` calls intended to be commented out? 

Yes. I heard today that we need more tests, so...

> If they aren't needed let's remove them. If they are intended to be commented out let's add a comment saying why.

It was commented out, because iOS does not work and I don't have time to investigate right now.

And maybe solution for some problems is not THAT far.

Reply:
Few days ago we heard that we need more testing and I think more samples do fall under that category (more testing).

Here

https://github.com/xamarin/AndroidX/pull/575/files

you just split one solution into two.

I added MAUI, Xamarin.Forms samples and tried to resurrect old samples that were there for quite some time. 

... and I lost too much time, so I commented out few of them. I created separate repo to try to fix some of the attempts and will come back to fix and enhance samples.

And the greatest problem of them all is commented code.